docs/releases/1.8.3.txt

```diff
@@ -50,3 +50,6 @@ Bugfixes
 
 * Fixed recording of applied status for squashed (replacement) migrations
   (:ticket:`24628`).
+
+* Fixed prefetch_related for models using UUID primary keys on non-PostgreSQL
```

Comment:
Apparently I may be wrong here, not sure. I'm sure @timgraham the docs master will know.